dev/CommandBarFlyout/CommandBarFlyoutCommandBar.idl

```diff
@@ -27,6 +27,7 @@ runtimeclass CommandBarFlyoutCommandBarTemplateSettings
     Double ExpandDownAnimationHoldPosition{ get; };
     Windows.Foundation.Rect ContentClipRect { get; };
     Windows.Foundation.Rect OverflowContentClipRect{ get; };
+    Windows.Foundation.Rect AcrylicBackgroundBaseClipRect{ get; };
```

Comment:
What is this property for? It doesn't seem to be used anywhere.

Reply:
haha, you caught this before I pushed a change 😂